src/ensembl/src/shared/helpers/urlHelper.ts

```diff
@@ -1,3 +1,4 @@
-export const speciesSelector = () => '/app/speicies-selector';
+export const speciesSelector = () => '/app/species-selector';
 export const customDownload = () => '/app/custom-download';
-export const browser = () => '/app/browser';
+export const browser = (genomeId: string, focus: string, location: string) =>
+  `/app/browser/${genomeId}?focus=${focus}&location=${location}`;
```

Comment:
This can’t be right. It should be possible to generate an "empty" genome browser url (/app/browser) that is going to show a screen with example objects for the first selected species. I suggest the browser url builder receive an optional params object with genomeId, focus and location properties.

Reply:
I see @msmanoj has fixed this. So I have marked this as resolved.